website/README.md

```diff
@@ -31,6 +32,43 @@ If you find a typo or you feel like you can improve the HTML, CSS, or JavaScript
 
 <!-- END: contributions -->
 
+## Backport labels
+
+The following backport labels exist to help facilitate backporting
+changes to `stable-website` and or `release-#.#.#` branches.
+
+### `backport/website`
+
+Merging a PR with this label will trigger two[^backport-strategy] workflow jobs:
+
+1. Attempt to backport changes into `stable-website`
```

Comment:
It might help to clarify that `stable-website` represents the same version of the docs as `release-#.#.#` , and not `main`.

Reply:
Included a note